pkg/controller/storagecluster/testspec/storkSchedDeployment.yaml

```diff
@@ -36,7 +36,7 @@ spec:
         - --policy-configmap=stork-config
         - --policy-configmap-namespace=kube-test
         - --lock-object-name=stork-scheduler
-        image: gcr.io/google_containers/kube-scheduler-amd64:v1.12.0
+        image: gcr.io/google_containers/kube-scheduler-amd64:v1.21.0
```

Comment:
you can change this to `k8s.gcr.io/kube-scheduler-amd64:v1.21.0`

Reply:
We have test cases on older versions like ```v1.17.0```, which looks for image at ```gcr.io/google_containers/kube-scheduler-amd64:v1.17.0```, shall we create a different deployment spec for testing versions 1.21 and 1.22?